Source/Core/DolphinQt/Achievements/AchievementSettingsWidget.cpp

```diff
@@ -36,7 +36,7 @@ AchievementSettingsWidget::AchievementSettingsWidget(QWidget* parent) : QWidget(
 
   // If hardcore is enabled when the emulator starts, make sure it turns off what it needs to
   if (Config::Get(Config::RA_HARDCORE_ENABLED))
-    ToggleHardcore();
+    emit Settings::Instance().HardcoreStateChanged();
```

Comment:
This problem exists since before this PR, but relying on DolphinQt to call the core's SetHardcoreMode function is a fragile design. If the user is running a frontend other than DolphinQt, hardcode mode won't be applied. This would make more sense to put in UICommon or in AchievementManager.